core/controllers/reader.py

```diff
@@ -55,6 +55,17 @@
 def require_playable(handler):
     """Decorator that checks if the user can play the given exploration."""
     def test_can_play(self, exploration_id, **kwargs):
+        """
+        Args:
+            exploration_id: Id of the exploration
+            kwargs: Any other keyworded args.
+
+        Returns:
+            either None if exploration_id is in disabled ids or
+            a handler instance with updated args.
+        Raises:
+            PageNotFound if user is not logged in.
```

Comment:
Formatting should be:

`PageNotFoundException: The user is not logged in.`

That said: is the above the only case where a PageNotFoundException might be raised?